File: python/user_link.py
from linebot import (
    LineBotApi,
)

line_bot_api = LineBotApi('LINE_CHANNEL_TOKEN') # 上書きする

# === リッチメニューの情報を見る ===
for rich_menus in line_bot_api.get_rich_menu_list():
  print(rich_menus.rich_menu_id)

user_id = 'Uxxxxxx'
richmenu_id = 'richmenu-xxx'

# # === 個別リッチメニューリンク ===
# line_bot_api.link_rich_menu_to_user(user_id, richmenu_id)
# print(line_bot_api.get_rich_menu_id_of_user(user_id))
# line_bot_api.unlink_rich_menu_from_user(user_id)

# # === 一括リッチメニューリンク ===
# line_bot_api.link_rich_menu_to_users([user_id], richmenu_id)
# print(line_bot_api.get_rich_menu_id_of_user(user_id))
# line_bot_api.unlink_rich_menu_from_users([user_id])

File: node/user_link.js
// CommonJS
const line = require('@line/bot-sdk');

const config = {
  channelAccessToken: 'LINE_CHANNEL_TOKEN', // 上書きする
  channelSecret: 'LINE_CHANNEL_SECRET'      // 上書きする
};
client = new line.Client(config);

const main = async (client) => {
  // === リッチメニューの情報を見る
  await client.getRichMenuList().then((r) => {
    console.log(r)
  })

  userId = 'Uxxxxxx'
  richmenuId = 'richmenu-xxx'

  // // === 個別リッチメニューリンク ===
  // await client.linkRichMenuToUser(userId, richmenuId).then(() => {}, () => {})
  // await client.getRichMenuIdOfUser(userId).then((r) => {
  //   console.log(r)
  // })
  // await client.unlinkRichMenuFromUser(userId)

  // // === 一括リッチメニューリンク ===
  // await client.linkRichMenuToMultipleUsers(richmenuId, [userId]).then(() => {}, () => {})
  // await client.getRichMenuIdOfUser(userId).then((r) => {
  //   console.log(r)
  // })
  // await client.unlinkRichMenusFromMultipleUsers([userId])
}
main(client)
